### PR TITLE
SDK-2024: Updated manifest types documentation

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/SDK-2028

### CODE REVIEW

File: docs/manifest-types.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Manifest types
 
+[Public documentation]([Public documentation](https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/code-upgrader-supported-extension-scenarios.html)) for the section
```
Comment (DmytroKlymanSpryker) [inline docs/manifest-types.md:3]: I don't understand this sentence to be honest: 

"Public documentation for the section".

Maybe it should be something like "Description of the supported extension scenarios can be found [here]" or something like that.

Comment (VladislavStrelchenko) [reply docs/manifest-types.md]: Thanks 
Changed these message

File: docs/manifest-types.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ For common types, all manifests have the following structure.
 Generation is currently handled internally in the Spryker release app, whereas integration is done through this code base directly with
 [Strategy classes](//github.com/spryker-sdk/integrator/tree/master/src/ManifestStrategy/).
 
+Description of the supported extension scenarios can be found [here](https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/code-upgrader-supported-extension-scenarios.md)
```
Comment (DmytroKlymanSpryker) [inline docs/manifest-types.md:53]: .md -> .html

Comment (VladislavStrelchenko) [reply docs/manifest-types.md]: ++

**Review by DmytroKlymanSpryker — APPROVED** [review]

**Comment by DmytroKlymanSpryker** [comment]
Should be released after: https://github.com/spryker/spryker-docs/pull/1675

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**